src/map/ai/helpers/gambits_container.cpp

```diff
@@ -656,6 +656,16 @@ namespace gambits
                 return noStorm;
                 break;
             }
+            case G_CONDITION::PT_HAS_TANK:
+            {
+                return PartyHasTank();
+                break;
+            }
+            case G_CONDITION::NOT_PT_HAS_TANK:
+            {
+                return (!PartyHasTank());
```

Comment:
This does need parens around it

Reply:
Thanks for the review!

I'm a bit confused, would you mind clarifying this statement? Is the suggestion to wrap the return in parens (e.g.): 

`(return (!PartyHasTank()));`

or something else?

Reply:
Just this:

`return !PartyHasTank();`